.travis.yml

```diff
@@ -22,6 +25,7 @@ env:
    - secure: "OKulfkA5OGd/d1IhvBKzRkHQwMcWjzrzbimo7+5NhkUkWxndAzl+719TB3wWvIh1i2wXXrEXsyZkXM5FtRrHm55v1VKQ5ibjEvFg1w3NIg81iDyoLq186fLqywvxGkOAFPrsePPsBj5USd5xvhwwbrjO6L7/RK6Z8shBwOSc41s="
 
 before_install:
+  - export CARGO_TARGET_DIR="$TRAVIS_BUILD_DIR/target"
```

Comment:
Could you remove `CARGO_TARGET_DIR` from `ci/base-tests.sh`?

Reply:
Done.